types/templates.go

```diff
@@ -1077,6 +1078,15 @@ type ValidatorHistory struct {
 	WithdrawalSlot    sql.NullInt64                `db:"withdrawal_slot" json:"withdrawal_slot,omitempty"`
 }
 
+type ValidatorDuty struct {
+	SourceAttestationStatus uint64
+	TargetAttestationStatus uint64
+	HeadAttestationStatus   uint64
+	BlockProposalStatus     uint64
+	SyncCommitteeStatus     uint64
+	SlashingStatus          uint64
+}
+
```

Comment:
This is not used and can be deleted.

Reply:
oversight sorry, fixed in new commit, ptal